CI/wininstaller/build_installer.cmd

```diff
@@ -38,6 +37,15 @@ if exist "%WinDir%\SysWow64" (
     set "ProgFiles=%programfiles%"
 )
 
+REM Dynamically locate the UCRT path
+set "UCRTBasePath=%ProgFiles%\Windows Kits\10\Redist"
+set "UCRTFilesPath="
+for /d %%d in ("%UCRTBasePath%\*") do (
+    if exist "%%d\ucrt\DLLs" (

```

Comment:
it should use user-provided directory name instead of selecting on its own. CMake has [CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION](https://cmake.org/cmake/help/latest/variable/CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION.html) with this info.

Reply:
I don't think Visual Studio users use CMake instead of "hit play" button. But I have modded script to have easy ability to provide custom folder. For example I have installed 10.0.19041.0 on my testing machine.

Reply:
visual studio project is generated by cmake, so... :)

later we'll adjust the installer script to pull info from cmake (probably just read some prepared file with directory name)

Reply:
still, your script pulls a random folder from WDK which is incorrect, as one can have multiple WDK installations

Reply:
ISS script takes params correctly from ISCC.exe, can be used directly as fheroes2 have or if is needed to use this CMD script in CI it should be modded them to accept parameters itsefl.
But that CI is out of my knowledge.. :)

And if you wanna to implement some add_custom_command to CMakeList then it will be needed if wanna to build installer automatically.

Reply:
> still, your script pulls a random folder from WDK which is incorrect, as one can have multiple WDK installations. 
Not random, latest available under Windows 10 SDK.

Reply:
Now it correctly detect latest Windows 10 SDK in any cases. Changed that dir loop for detection to be sure.